Formula/pixie.rb

```diff
@@ -7,6 +7,11 @@ class Pixie < Formula
   license "Apache-2.0"
   head "https://github.com/pixie-io/pixie.git", branch: "main"
 
+  livecheck do
+    url :stable
+    regex(%r{^releases/cli/v?(\d+(?:\.\d+)+)$}i)
```

Comment:
```suggestion
    regex(%r{^release/cli/v?(\d+(?:\.\d+)+)$}i)
```

The tag format is `release/cli/...` rather than `releases/cli/...`.